h2o-core/src/main/java/water/util/BytesStats.java

```diff
@@ -0,0 +1,78 @@
+package water.util;
+
+import java.io.Serializable;
+import water.util.Java7.Objects;
+
+/**
+ * Gathers stats for a byte array.
+ * 
+ * The array has to contain at least one NewLine character,
+ * otherwise it's considered bad, bad.
+ * 
+ * Created by vpatryshev on 3/27/17.
+ */
+public class BytesStats implements Serializable {
+  public final int numLines;
+  public final int maxWidth;
```

Comment:
What does maxWidth mean? How does it handle international (non-ascii) characters?